sources/packages/forms/src/form-definitions/sfaa2021-22.json

```diff
@@ -551,7 +551,8 @@
           "allowMultipleMasks": false,
           "inputType": "hidden",
           "id": "elknlor",
-          "addons": []
+          "addons": [],
```

Comment:
I could see differences that are different in this file and the rest of the files of sfaaXXXX-XX.json? Normally we used to match the same changes between the files until there is a change respect to program year. Can you please verify there is differences in the changes?

Reply:
If I understood @dheepak-aot 's explanation correctly, since formio added the "tags": [] line, it should be left there.